octodns_scaleway/__init__.py

```diff
@@ -2,20 +2,37 @@
 #
 #
 
+from enum import Enum
 from collections import defaultdict
 from requests import Session
 from logging import getLogger
 from urllib.parse import urlparse
 
 from octodns.record import Record
+from octodns.record.ds import DsValue
 from octodns.record.geo import GeoCodes
 from octodns.provider import ProviderException
 from octodns.provider.base import BaseProvider
 
-__VERSION__ = '0.0.4'
+__VERSION__ = '0.1.0'
 __API_VERSION__ = 'v2beta1'
 
 
+class ScalewayDSAlgorithm(Enum):
```

Comment:
I'm missing some dnssec algorithm (4, 9, 11) https://www.iana.org/assignments/dns-sec-alg-numbers/dns-sec-alg-numbers.xhtml

Does the documentation is incorrect or it's just unsupported ?
https://www.scaleway.com/en/developers/api/domains-and-dns/registrar-api/#path-domains-update-domain-dnssec